affiliategw-message-handler/affiliategw-messaging-handler/app.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+from typing import Any, Dict
+
+import requests
+from chalice import Chalice
+from chalice.app import SQSEvent
+from requests.models import Response
+
+app = Chalice(app_name='affiliategw-messaging-handler')
+
+
+sentry_dsn = os.environ.get('affiliategw_messaging_handler_sentry_dsn')
+if sentry_dsn:
+    import sentry_sdk
+    from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+    sentry_sdk.init(
+        dsn=sentry_dsn,
+        integrations=[AwsLambdaIntegration()]
+    )
+
+
+QUEUE: str = 'affiliategw-tune-queue'
+TARGET_URL: str = 'https://ncotflvio7.execute-api.eu-west-1.amazonaws.com/api/'
+
+
+@app.on_sqs_message(queue=QUEUE, batch_size=1)  # type: ignore
+def index(sqs_event: SQSEvent) -> None:
+    """
+    Receives an SQS message and send it as an https payload.
+    """
+    handler(sqs_event)
```

Comment:
Should `usable_body` be extract from event here? Then handler function would be independent of Chalice framework.